angular/src/app/modules/delivery-management/delivery/available-resource-tab/will-pool/will-pool.component.ts

```
@@ -0,0 +1,386 @@
+        ? a.totalContribute - b.totalContribute
+        : b.totalContribute - a.totalContribute;
+    });
+  }
+
+  sortDataByEndChargeDate(direction: boolean): void {
+    const defaultGetTime = 1;
+    this.listWillPool.forEach(item => {
+      item.accounts.sort((a, b) => {
+        const getValidTimestamp = (dateString: string): number => {
+          const date = new Date(dateString);
+          return isNaN(date.getTime()) ? defaultGetTime : date.getTime();
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)